src/SFA.DAS.Data.Worker/Mapping/EventsMapping.cs

```diff
@@ -8,7 +9,11 @@ public class EventsMapping : Profile
     {
         public EventsMapping()
         {
-            CreateMap<ApprenticeshipEventView, CommitmentsApprenticeshipEvent>();
+            CreateMap<ApprenticeshipEventView, CommitmentsApprenticeshipEvent>()
+                .ForMember(target => target.PaymentStatus, 
+                    config => config.MapFrom(source => Enum.GetName(typeof(PaymentStatus), source.PaymentStatus)))
+                .ForMember(target => target.AgreementStatus,
+                    config => config.MapFrom(source => Enum.GetName(typeof(AgreementStatus), source.AgreementStatus)));
```

Comment:
I had a discussion with Simon and Dan about what should and should not be unit tested given the complexity of the code and I think I just originally deemed this to be too simple to unit test however I can created tests if you think its not. I'm happy either way and tbh may have unit tested it on a different day ( I just deemed it to simple at the time of writing it)